doc/cephadm/services/monitoring.rst

```diff
@@ -408,14 +408,17 @@ Configuring SSL/TLS for Grafana
 
 ``cephadm`` deploys Grafana using the certificate defined in the ceph
 key/value store. If no certificate is specified, ``cephadm`` generates a
-self-signed certificate during the deployment of the Grafana service.
+self-signed certificate during the deployment of the Grafana service. Each
+certificate is specific for the host it was generated on.
 
 A custom certificate can be configured using the following commands:
 
 .. prompt:: bash #
 
-  ceph config-key set mgr/cephadm/grafana_key -i $PWD/key.pem
-  ceph config-key set mgr/cephadm/grafana_crt -i $PWD/certificate.pem
+  ceph config-key set mgr/cephadm/{hostname}/grafana_key -i $PWD/key.pem
+  ceph config-key set mgr/cephadm/{hostname}/grafana_crt -i $PWD/certificate.pem
+
+Where `hostname` is the short hostname for the host where grafana service is deployed.
```

Comment:
Is this actually always the short hostname or is it just the hostname provided when adding the host with cephadm  that shows up in `orch host ls`?

Reply:
> Is this actually always the short hostname or is it just the hostname provided when adding the host with cephadm that shows up in `orch host ls`?

Good question, it's what we could get from `{daemon_spec.host}` could this be also an IP addr?

Reply:
I think it should be the hostname cephadm has stored. It either comes from the scheduler which just has the hostnames we pass it or from the daemon description, which should match the orch ps output.

Reply:
fixed